src/RuleTicket.php

```diff
@@ -177,6 +177,10 @@ public function getCriterias()
         $criterias['locations_id']['linkfield']               = 'locations_id';
         $criterias['locations_id']['type']                    = 'dropdown';
 
+        $criterias['locations_id_code']['table']              = 'glpi_locations';
```

Comment:
1. `_` is usually used for "virtual" fields.
2. `id` should not be part of the name, as it is not an id.
```suggestion
        $criterias['_location_code']['table']              = 'glpi_locations';
```